packages/cxl-ui/src/components/cxl-icon-nav-item.js

```diff
@@ -0,0 +1,32 @@
+// Import the LitElement base class and html helper function
+import { LitElement, html } from 'lit-element';
+import '@vaadin/vaadin-item';
+import cxlThemeStyles from '../styles/cxl-theme-css.js';
+import cxlIconNavItemStyles from '../styles/cxl-icon-nav-item-css.js';
+
+class CXLIconNavItemElement extends LitElement {
+  static get properties() {
+    return {
+      href: {
+        type: String
+      },
+      icon: {
+        type: String
+      }
+    };
+  }
+
+  static get styles() {
+    return [cxlThemeStyles, cxlIconNavItemStyles];
+  }
+
+  render() {
+    return html`
+      <a href="${this.icon}">
+        <iron-icon class="icon size-l" icon="${this.icon}"></iron-icon>
```

Comment:
Need `active` state implementation.

Reply:
WP will output a set of relevant classes on `<vaadin-tab>`, it's too much work to re-architect that, so we should try to make it work.

Discuss when we come to it.